test/integration_tests/test_huggingface_udfs_ner.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+import pytest
+
+from eva.catalog.catalog_manager import CatalogManager
+from eva.server.command_handler import execute_query_fetch_all
+
+NUM_FRAMES = 10
+
+
+class HuggingFaceNERTests(unittest.TestCase):
+    """
+    The tests below essentially check for the output format returned by HF.
+    We need to ensure that it is in the format that we expect.
+    """
+
+    def setUp(self) -> None:
+        CatalogManager().reset()
+
+        query = """LOAD PDF 'data/documents/pdf_sample1.pdf' INTO MyPDFs;"""
+        execute_query_fetch_all(query)
+
+    def tearDown(self) -> None:
+        execute_query_fetch_all("DROP TABLE IF EXISTS MyPDFs;")
+
+    @pytest.mark.benchmark
+    def test_named_entity_recognition_model(self):
```

Comment:
Also add a test case for when there are no named entities found in a text document. We want to ensure that EVA exits gracefully in that case.

Reply:
made this changes and pushed and the test case name is test_named_entity_recognition_model_no_ner_data_exists